### ISSUE TITLE
Windows installer 2017.7.2 missing libnacl

### ISSUE THREAD

**Issue by mbochenk** [saltstack/salt#45178]
### Setup
Generate a windows VM via salt-cloud, add solt via bootstrap.

### Steps to Reproduce Issue
Execute anything that uses salt.nacl - salt.acl.dec, etc. For ex. render a file with `superpassword={{salt.nacl.dec(pg_password)}}`

Error in logs:
```
[DEBUG   ] Could not LazyLoad nacl.dec: 'nacl' __virtual__ returned False: libnacl import error, perhaps missing python libnacl package
[ERROR   ] Rendering exception occurred: Jinja variable 'salt.utils.templates.AliasedLoader object' has no attribute 'nacl.dec'
[CRITICAL] Rendering SLS 'dev:core-software.databases.postgresql.win-pgsql' failed: Jinja variable 'salt.utils.templates.AliasedLoader object' has no attribute 'nacl.dec'
```

If pip shipped with salt used to install libnacl - `pip2 install libnacl` - everything works again

### Versions Report
Master is on latest CentOS 7 and salt 2017.7.2. It also seems to affect Py3 versions, however I did not look much into them.

Minion (windows version does not matter, after installing via pip libnacl is reported as installed):
```
Salt Version:
           Salt: 2017.7.2

Dependency Versions:
           cffi: 1.10.0
       cherrypy: 10.2.1
       dateutil: 2.6.0
      docker-py: Not Installed
          gitdb: 2.0.3
      gitpython: 2.1.3
          ioflo: Not Installed
         Jinja2: 2.9.6
        libgit2: Not Installed
        libnacl: Not Installed
       M2Crypto: Not Installed
           Mako: 1.0.6
   msgpack-pure: Not Installed
 msgpack-python: 0.4.8
   mysql-python: Not Installed
      pycparser: 2.17
       pycrypto: 2.6.1
   pycryptodome: Not Installed
         pygit2: Not Installed
         Python: 2.7.13 (v2.7.13:a06454b1afa1, Dec 17 2016, 20:53:40) [MSC v.150
0 64 bit (AMD64)]
   python-gnupg: 0.4.0
         PyYAML: 3.11
          PyZMQ: 16.0.2
           RAET: Not Installed
          smmap: 2.0.3
        timelib: 0.2.4
        Tornado: 4.5.1
            ZMQ: 4.1.6

System Versions:
           dist:
         locale: cp1252
        machine: AMD64
        release: 2012ServerR2
         system: Windows
        version: 2012ServerR2 6.3.9600  Multiprocessor Free
```

**Comment by garethgreenaway** [comment]
@twangboy @dmurphy18 FYI

**Comment by mbochenk** [comment]
Seems to have been removed again by https://github.com/saltstack/salt/pull/46990/commits/b9efec8526f323bdc27350104a8be973752221a4

**Comment by mbochenk** [comment]
@twangboy It seems this is merged to 2017.7, thanks! FYI deps were also removed from 2018.3

**Comment by twangboy** [comment]
@mbochenk These changes will be merged forward to 2018.3.
